jans-linux-setup/jans_setup/setup_app/utils/package_utils.py

```
@@ -127,17 +129,21 @@ def check_and_install_packages(self):
                 if install[install_type]:
                     self.logIt("Installing packages " + packages)
                     print("Installing packages", packages)
-                    if not base.os_type in ('fedora', 'suse'):
+                    if base.os_type not in ('suse',):
                         sout, serr = self.run(update_command, shell=True, get_stderr=True)
```

Ruff (0.15.9)

[warning] Unpacked variable `sout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)